README.md

```diff
@@ -1,8 +1,10 @@
+
 ---
 
 :warning: This repository will be archived in November 2022 as it is no longer in development. :warning:
```

Comment:
Just wondering if you intended the :warning: to be bold or italic such as ***warning*** instead?

Reply:
Ah interesting so 'warning' surrounded by a pair of ':'  gives a nice little 'attention' icon. I don't see that in my ide!